Upstream: No (considered finished)
Reason: wrong `errno` (see below)
Source: Daniel Mierswa <impulze@impulze.org>

Compilation failure:
	ld: errno: TLS definition in /lib64/libc.so.6 section .tbss mismatches non-TLS reference in *.o

--- ./error.h.orig	2009-01-28 18:11:38.788975742 +0100
+++ ./error.h	2009-01-28 18:11:15.614974533 +0100
@@ -1,7 +1,7 @@
 #ifndef ERROR_H
 #define ERROR_H
 
-extern int errno;
+#include <errno.h>
 
 extern int error_intr;
 extern int error_nomem;
